registry/server/migrations/20210505151225_routes_domainIdIdxble.ts

```diff
@@ -0,0 +1,25 @@
+import * as Knex from "knex";
+import isMySQL from "../util/isMySQL";
+
+
+export async function up(knex: Knex): Promise<void> {
+    if (isMySQL(knex)) {
+        return knex.raw("ALTER TABLE routes ADD domainIdIdxble int(11) GENERATED ALWAYS AS (coalesce(domainId, -1)) STORED;");
+    } else {
+        return knex.raw("ALTER TABLE routes ADD COLUMN domainIdIdxble INT GENERATED ALWAYS AS (coalesce(domainId, -1)) VIRTUAL;");
```

Comment:
do you use "VIRTUAL" since it's the only way to alter a table in sqlite?

Reply:
yep